CHANGELOG.md

```diff
@@ -7,13 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- `collection_items` to request collection items through a STAC API
```

Comment:
hmm, maybe you should rebase on latest master (or merge master), there should be already "added" entries here

Reply:
Done